indico/modules/events/abstracts/templates/management/_abstract_list.html

```diff
@@ -56,7 +56,10 @@
                             {% for item in dynamic_columns %}
                                 <th class="i-table">{{ item.title }}</th>
                             {% endfor %}
-                            <th class="i-table thin-column"></th>
+                            {% if reviewed_abstracts %}
```

Comment:
Cause otherwise the column does not serve any purpose. If most of the team agrees on adding the column even if it's empty, I'll gladly do it! @pferreir ?